### ISSUE TITLE
Geo component and concept role name

### ISSUE THREAD

**Issue by achronis** [sdmx-twg/sdmx-ml#26]
Hi

(in case it is not fixed already)
It is about an inconsistency between public draft Technical Notes and a sample here in sdmx-ml concerning the concept role id.
(relates to  https://github.com/sdmx-twg/sdmx-ml/issues/9 )

The [Technical Notes line 1172](https://sdmx.org/wp-content/uploads/SDMX_3-0-0_SECTION_6_DRAFT-1-0.pdf) notes that the role is `GEO_FEATURE_SET`:

> 1171 Any Component used for representing a Geographical Feature Set, i.e., used to
> 1172 describe geographical characteristics, must have a “GEO_FEATURE_SET” role. Its
> 1173 Representation would be of textType="GeospatialInformation". 

But in the sdmx-ml sample [samples/Geospatial/geospatial_geocomponents.xml#L58](https://github.com/sdmx-twg/sdmx-ml/blob/develop/samples/Geospatial/geospatial_geocomponents.xml#L58) it has role `GEO`.

**Comment by dosse** [comment]
Hi @DrJMunozMx, @jgager could you please help answering this question?

**Comment by dosse** [comment]
@dbarr99 Currently, the [global SDMX Concept Scheme for Concept Roles](https://registry.sdmx.org/sdmx/v2/structure/conceptscheme/SDMX/SDMX_CONCEPT_ROLES/1.0) only has a concept for `GEO`, but none for the new role `GEO_FEATURE_SET` as defined in SDMX 3.0 in the [SDMX Technical Notes](https://sdmx.org/wp-content/uploads/SDMX_3-0-0_SECTION_6_DRAFT-1-0.pdf), line 1172:  
> 1171 Any Component used for representing a Geographical Feature Set, i.e., used to
> 1172 describe geographical characteristics, must have a “GEO_FEATURE_SET” role. Its
> 1173 Representation would be of textType="GeospatialInformation".

Are you planning to add this role? We can then update the [samples/Geospatial/geospatial_geocomponents.xml](https://github.com/sdmx-twg/sdmx-ml/blob/develop/samples/Geospatial/geospatial_geocomponents.xml).

**Comment by hoehrmann** [comment]
I ended up here because https://sdmx.org/wp-content/uploads/SDMX_3-0-0_SECTION_2_FINAL-1_0.pdf has a ComponentRoleType with `geo_feature_set` which I could not find anywhere but here.

**Comment by DrJMunozMx** [comment]
Yes, the roles list must be updated in the COG. By the moment we have the example using the GEO role as a temporary way to represent it.
